client/coral-embed-stream/src/containers/Embed.js

```diff
@@ -20,12 +22,72 @@ const {logout, checkLogin} = authActions;
 const {fetchAssetSuccess} = assetActions;
 
 class EmbedContainer extends React.Component {
+  subscriptions = [];
+
+  subscribeToUpdates(props = this.props) {
+    if (props.auth.loggedIn) {
+      let sub = props.data.subscribeToMore({
+        document: USER_BANNED_SUBSCRIPTION,
+        variables: {
+          user_id: props.auth.user.id,
+        },
+        updateQuery: () => {
+          addNotification('info', t('your_account_has_been_banned'));
+        },
+      });
+
+      this.subscriptions.push(sub);
+
+      sub = props.data.subscribeToMore({
+        document: USER_SUSPENDED_SUBSCRIPTION,
+        variables: {
+          user_id: props.auth.user.id,
+        },
+        updateQuery: () => {
+          addNotification('info', t('your_account_has_been_suspended'));
+        },
+      });
+
+      this.subscriptions.push(sub);
+
+      sub = props.data.subscribeToMore({
+        document: USERNAME_REJECTED_SUBSCRIPTION,
+        variables: {
+          user_id: props.auth.user.id,
+        },
+        updateQuery: () => {
+          addNotification('info', t('your_username_has_been_rejected'));
+        },
+      });
+
+      this.subscriptions.push(sub);
+    }
+  }
```

Comment:
Thanks @ALawliet, I love your first suggestion :-) I find the second one a bit too rigid. 👍 💯